lib/msf/core/db_manager/http/sinatra_app.rb

```diff
@@ -17,11 +18,20 @@
 require 'msf/core/db_manager/http/servlet/db_export_servlet'
 require 'msf/core/db_manager/http/servlet/vuln_attempt_servlet'
 
+require 'swagger/blocks'
+
 class SinatraApp < Sinatra::Base
+  set :public_folder, File.dirname(__FILE__) + '/public'
```

Comment:
Setting `:public_folder` in this manner is unnecessary as it defaults to public. See [Sinatra - Configuring Settings](http://sinatrarb.com/configuration.html)
> By default, this is assumed to be a directory named “public” within the root directory